sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -122,6 +122,18 @@ namespace Azure { namespace Core { namespace Http {
      */
     std::string CAInfo;
 
+    /**
+     * @brief Path to a directory which holds PEM encoded file, containing the certificate
+     * authorities sent to libcurl handle directly.
+     *
+     * @remark The Azure SDK will not check if the path is valid or not.
+     *
+     * @remark The default is the built-in system specific path. More about this option:
+     * https://curl.se/libcurl/c/CURLOPT_CAPATH.html
```

Comment:
The CURL documentation for this option says that it doesn't work on WIndows. When building the SDK for windows, this option should either be conditionally compiled out, or should throw an exception if the CAPath value is non-empty.

Reply:
Agreed. Macro `SUPPORT_SETTING_CAPATH` is added for the conditional compilation.